app/upgrades/v7/upgrades.go

```diff
@@ -132,6 +139,56 @@ func auctionInitGenesis(ctx sdk.Context, auctionKeeper auctionkeeper.Keeper) {
 	auctionkeeper.InitGenesis(ctx, auctionKeeper, genesisState)
 }
 
+// Initialize the Axelar cork module with the correct parameters.
+func axelarcorkInitGenesis(ctx sdk.Context, axelarcorkKeeper axelarcorkkeeper.Keeper) {
+	genesisState := axelarcorktypes.DefaultGenesisState()
+
+	genesisState.Params.Enabled = true
+	genesisState.Params.TimeoutDuration = uint64(6 * time.Hour)
+	genesisState.Params.IbcChannel = "channel-5"
+	genesisState.Params.IbcPort = ibctransfertypes.PortID
+	genesisState.Params.GmpAccount = "axelar1dv4u5k73pzqrxlzujxg3qp8kvc3pje7jtdvu72npnt5zhq05ejcsn5qme5s"
+	genesisState.Params.ExecutorAccount = "axelar1aythygn6z5thymj6tmzfwekzh05ewg3l7d6y89"
+	genesisState.Params.CorkTimeoutBlocks = 5000
+
+	genesisState.ChainConfigurations = axelarcorktypes.ChainConfigurations{
+		Configurations: []*axelarcorktypes.ChainConfiguration{
+			{
+				Name:         "arbitrum",
+				Id:           42161,
+				ProxyAddress: "0xEe75bA2C81C04DcA4b0ED6d1B7077c188FEde4d2",
```

Comment:
There's no targeting of gateways. The proxy contract is the contract we're trying to hit that is an Axelar executable. The gateway reaches out itself to call the executable (calling the execute function on the proxy contract) when the call is recorded in state on Axelar.

Reply:
The addresses are the same because Crispy deterministically deployed it to each L2.

Reply:
> The addresses are the same because Crispy deterministically deployed it to each L2.

Is there anything expecting the casing checksum to match the respective chains?

Reply:
I'm not certain, we're passing along a string in the ICS-20 message and Axelar is handling it. I assume they are resolving it correctly.

Reply:
Eth doesn't include chain information in the checksums for addresses like Cosmos does